src/fs/rock/RockRebuild.cc

```diff
@@ -146,7 +193,7 @@ class LoadingSlot
 class LoadingParts
 {
 public:
-    LoadingParts(int dbSlotLimit, int dbEntryLimit);
+    LoadingParts(const char *dirPath);
     LoadingParts(LoadingParts&&) = delete; // paranoid (often too huge to copy)
```

Comment:
That explanation is no longer true because this class now stores just pointers. Perhaps we should delete this explicit copying/moving prohibition now?